lib/request-handler.js

```diff
@@ -64,12 +76,18 @@ module.exports = function processRequest(options, request, response) {
 
     const handleError = err => {
         this.emit('error', request, err);
+        span.setTag(Tags.ERROR, true);
+        span.log({
+            message: err.presentable,
```

Comment:
the presentable info can sometimes be a default template, should we log this?

Reply:
Most probably not. I don't have enough knowledge about those details. Please suggest what you believe is more reasonable